backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -458,6 +461,47 @@ public TestEvent correctTestMarkAsError(UUID testEventId, String reasonForCorrec
     return newRemoveEvent;
   }
 
+  @Transactional(readOnly = true)
+  @AuthorizationConfiguration.RequirePermissionEditOrganization
+  public OrganizationLevelDashboardMetrics getOrganizationLevelDashboardMetrics(
+      Date startDate, Date endDate) {
+    Organization org = _os.getCurrentOrganization();
+    List<UUID> facilityIds =
+        _os.getFacilities(org).stream().map(Facility::getInternalId).collect(Collectors.toList());
+
+    List<AggregateFacilityMetrics> facilityMetrics = new ArrayList<AggregateFacilityMetrics>();
+
+    for (UUID facilityId : facilityIds) {
+      List<TestResultWithCount> results =
+          _terepo.countByResultForFacility(facilityId, startDate, endDate);
+      Facility facility = _os.getFacilityInCurrentOrg(facilityId);
```

Comment:
We have some organizations with quite a few facilities.  Would it be difficult to get all of the data in one query instead of each facility individually?

Reply:
Tried hammering on this for a while to no avail :( It would certainly optimize the query to fetch everything at once and group by both facility and result type, but I'm going to shift that into a future ticket as I'd like to get at least a v0 of the facility table out for users.